lib/java/spring/events/src/main/java/co/airy/spring/events/custom/HttpEventPublisher.java

```diff
@@ -0,0 +1,20 @@
+package co.airy.spring.events.custom;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Component
+public class HttpEventPublisher {
+    @Autowired
```

Comment:
apart from test code we're not autowiring anymore so I'd use a final var with a constructor her as well